packages/next/test/integration/integration-1.test.js

```diff
@@ -2,15 +2,54 @@ process.env.NEXT_TELEMETRY_DISABLED = '1';
 
 const path = require('path');
 const fs = require('fs-extra');
-const builder = require('../../');
+const builder = require('../../dist');
```

Comment:
Was this intentional?

Reply:
It was not. Updated. Thanks!
[`3d0e1ce` (#9774)](https://github.com/vercel/vercel/pull/9774/commits/3d0e1cea2e7fd171aa865411ed1ffb8f10b43d98)